src/background/detect-content-type.ts

```diff
@@ -1,7 +1,9 @@
+type PDFContentType = { type: 'PDF' };
+
+type HTMLContentType = { type: 'HTML' };
```

Comment:
Since these used as part of an exported type then I think it makes sense to export them as well.